Docs/Book/Cookbook_v2.rst

```diff
@@ -0,0 +1,1017 @@
+RDKit Cookbook v2
+%%%%%%%%%%%%%%%%%%%%
+
+.. sectionauthor:: Vincent F. Scalfani <vfscalfani@ua.edu>
+
+Introduction
+**************
+
+What is this?
+===============
+
+This document provides example recipes of how to carry out particular tasks using the RDKit 
+functionality from Python. The contents have been contributed by the RDKit community, 
+tested with the latest RDKit release, and then compiled into this document. Note that this 
+document is the second iteration of the :doc:`Cookbook` (i.e., v2). The old Cookbook written 
+in Markdown is no longer maintained, but is available in prior RDKit releases for reference. 
+The RDKit Cookbook v2 is written in reStructuredText, which supports Sphinx doctests, 
+allowing for easier validation and maintenance of the RDKit Cookbook v2 code examples, where appropriate. 
+
+What gets included?
+=====================
+
+The examples included come from various online sources such as blogs, shared gists, and 
+the RDKit mailing lists. Generally, only minimal editing is added to the example code/notes for 
+formatting consistency and to incorporate the doctests. We have made a conscious effort 
+to appropriately credit the original source and authors. One of the first priorities of this
+document is to compile useful **short** examples shared on the RDKit mailing lists, as 
+these can be difficult to discover. It will take some time, but we hope to expand this 
+document into 100s of examples. As the document grows, it may make sense to prioritize 
+examples included in the RDKit Cookbook v2 based on community demand.
+
+Feedback and Contributing
+==========================
+
+If you have suggestions for how to improve the Cookbook v2 and/or examples you would like 
+included, please contribute directly in the source document (the .rst file).
+Alternatively, you can also send Cookbook revisions and addition requests to the mailing list:
+<rdkit-discuss@lists.sourceforge.net> (you will need to subscribe first).
+
+.. note::
+
+   The Index ID# (e.g., **RDKitCB_##**) is simply a way to track Cookbook entries and image file names. 
+   New Cookbook additions are sequentially index numbered, regardless of where they are placed 
+   within the document. As such, for reference, the next Cookbook entry is **RDKitCB_20**.
+
+Drawing Molecules (in a Jupyter Environment)
+**********************************************
+
+Include an Atom Index
+======================
+
+| **Author:** Takayuki Serizawa
+| **Source:** `<https://iwatobipen.wordpress.com/2017/02/25/draw-molecule-with-atom-index-in-rdkit/>`_
+| **Index ID#:** RDKitCB_0
+| **Summary:** Draw a molecule with atom index numbers.
+
+.. testcode::
+
+   from rdkit import Chem
+   from rdkit.Chem.Draw import IPythonConsole
```

Comment:
I'm not sure I would change the code, but when running a Jupyter notebook

```
from rdkit.Chem.Draw import IPythonConsole
```

is done automatically now.